src/lib/runner.rs

```diff
@@ -496,7 +496,11 @@ fn create_watch_task(task: &str, options: Option<TaskWatchOptions>) -> Task {
     task_config
 }
 
-fn create_proxy_task(task: &str, allow_private: bool, skip_init_end_tasks: bool) -> Task {
+pub(crate) fn create_proxy_task(
```

Comment:
runner is using execution_plan and now execution_plan is using runner.
this is cyclic dependency which i prefer to break if possible.
maybe we can move this function to another module? proxy.rs (probably bad name....)?